dev-util/scons/files/scons-3.0.5-jni.h-include-path.patch

```diff
@@ -0,0 +1,15 @@
+diff -Nur old/testing/framework/TestSCons.py new/testing/framework/TestSCons.py
```

Comment:
Similar to above, plus merge the two patches into one.

Reply:
The second patch is applicable only if use "test" while the first one affects "engine".
I'm not sure that changes of "engine" isn't required outside the test env.